spec/feature/queue/bulk_task_assignment_spec.rb

```diff
@@ -19,11 +19,11 @@ def fill_in_and_submit_bulk_assign_modal
       number_of_tasks = options.find { |option| option.text =~ /3/ }
```

Comment:
```suggestion
      number_of_tasks = options.find { |option| option.text =~ /3 \(all available tasks\)/ }
```
Seems to fix the flake.